### PR TITLE
Add zacfinance space

### PR DESCRIPTION
**Pull request by ZAC-FINANCE**
**Add space for ZACFinance**  
  
Etherscan: https://etherscan.io/token/0x98a90499b62Ae48E151a66B0F647570b5a473B1c
CoinGecko: https://www.coingecko.com/en/coins/zac-finance 
Website: https://zac.finance/  
Twitter: https://twitter.com/FinanceZac

### CODE REVIEW

File: spaces/zacfinance/index.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "token": "0x98a90499b62Ae48E151a66B0F647570b5a473B1c", 
+  "name": "ZAC.Finance", 
+  "network": "1", 
+  "symbol": "ZAC", 
+  "skin": "zacfinance",
+  "domain": "governance.zac.finance",
```
Comment (gokaiorg) [inline spaces/zacfinance/index.json:7]: If you want to add custom styles, please add your custom colours in `skins/zacfinance.scss` or delete `skin` field from `index.json` file to use default skin. 
If you want to add a custom domain, please add your custom domain in `spaces/domains.json` or delete `domain` field from `index.json` file.

**Review by gokaiorg — APPROVED** [review]

File: spaces/zacfinance/index.json (hunk 1 above)
Comment (ZAC-FINANCE) [reply spaces/zacfinance/index.json]: done, check again.

File: spaces/zacfinance/index.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "token": "0x98a90499b62Ae48E151a66B0F647570b5a473B1c", 
+  "name": "ZAC.Finance", 
+  "network": "1", 
+  "symbol": "ZAC", 
+  "skin": "use default skin",
+  "domain": "",
```
Comment (gokaiorg) [inline spaces/zacfinance/index.json:7]: Delete completely this 2 lines.

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/zacfinance/index.json (hunk 2 above)
Comment (ZAC-FINANCE) [reply spaces/zacfinance/index.json]: done, check again.

Thanks.

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**